linkerd/app/core/src/trace.rs

```diff
@@ -7,20 +7,32 @@ use tracing_subscriber::{
     reload, EnvFilter, FmtSubscriber,
 };
 
-const ENV_LOG: &str = "LINKERD2_PROXY_LOG";
-
-type Subscriber = Formatter<format::DefaultFields, format::Format<format::Full, Uptime>>;
+const ENV_LOG_LEVEL: &str = "LINKERD2_PROXY_LOG";
+const ENV_LOG_FORMAT: &str = "LINKERD2_PROXY_LOG_FORMAT";
```

Comment:
These should probably go in [`env.rs`](https://github.com/linkerd/linkerd2-proxy/blob/master/linkerd/app/src/env.rs), where we define all the other environment variables.

Reply:
I'm having particularly a lot of trouble with this.

After moving these and deifining their `DEFAULT_*` counterparts in env.rs, I started going down a rabbit hole of trying to do something similar to: https://github.com/linkerd/linkerd2-proxy/blob/a7f22f6caae7598855e8ae04d90e8668be6ca61b/linkerd/app/core/src/dns.rs#L6-L26 but it just seems like way too much plumbing.

How can I more simply just access the consts directly from env.rs? I tried importing app to try to get them via `app::env::ENV_LOG_LEVEL` for example but that resulted in a circular dependency error.

Reply:
In the meantime I've added the default values here in `trace.rs`  https://github.com/linkerd/linkerd2-proxy/pull/500/commits/337b7e63cc558f3ee8e3861a1e30a252846a2219